beets/autotag/mb.py

```diff
@@ -276,11 +276,22 @@ def album_info(release):
         disambig.append(release.get('disambiguation'))
     info.albumdisambig = u', '.join(disambig)
 
-    # Release type not always populated.
+    # Considers all release types (both primary and secondary) and
+    # stores them as a comma-separated string
+    all_types = []
+    if 'primary-type' in release['release-group']:
+        rel_primarytype = release['release-group']['primary-type']
+        if rel_primarytype:
+            all_types.append(rel_primarytype.lower())
+    if 'secondary-type-list' in release['release-group']:
+        all_types.extend([secondarytype.lower() for secondarytype in\
```

Comment:
It's currently complaining that the slash is there when it doesn't need to be ("backslash is redundant").